addons/mass_mailing/tests/test_mailing_mailing_schedule_date.py

```diff
@@ -31,3 +31,77 @@ def test_mailing_schedule_date(self):
         self.assertEqual(mailing.schedule_date, datetime(2021, 4, 30, 9, 0))
         self.assertEqual(mailing.schedule_type, 'scheduled')
         self.assertEqual(mailing.state, 'in_queue')
+
+    def test_mailing_next_departure(self):
```

Comment:
Note that you may use freezetime to use fake datetimes, and notably force "now" to be whatever you want. It allows notably to run some schedulers while faking the current datetime and check that conditions are well respected. This generally leads to more readable code :)

Reply:
Thanks for the info. Reworking the test and using your input!

Reply:
Added freeze_time approach on most recent commit!